collection-scripts/gather_network_logs

```diff
@@ -73,6 +74,48 @@ function gather_scale_data {
   echo pods amount: $(oc get pods -A | wc -l) >> "${NETWORK_LOG_PATH}"/cluster_scale & PIDS+=($!)
 }
 
+function gather_metallb_data {
+  echo "INFO: Gathering metallb data"
+  declare -a FILES_TO_GATHER=("frr.conf" "frr.log" "daemons" "vtysh.conf")
+  declare -a FRR_CONTAINERS=("frr" "reloader" "frr-metrics")
+  SPEAKER_PODS="${@:-$(oc -n ${METALLB_NS} get pods -l component=speaker -o jsonpath='{.items[*].metadata.name}')}"
+  METALLB_PATH=$NETWORK_LOG_PATH/${METALLB_NS}
+  
+  mkdir -p $METALLB_PATH
+
+  for SPEAKER_POD in ${SPEAKER_PODS[@]}; do
+    for CONTAINER in "${FRR_CONTAINERS[@]}"; do
+      mkdir -p $METALLB_PATH/pods/$SPEAKER_POD/${CONTAINER}/
+      for FILE in "${FILES_TO_GATHER[@]}"; do
+          oc -n ${METALLB_NS} exec ${SPEAKER_POD} -c ${CONTAINER} -- sh -c "cat /etc/frr/${FILE}" > \
+          ${METALLB_PATH}/pods/${SPEAKER_POD}/${CONTAINER}/${FILE}
+          
+          dump_frr_logs ${SPEAKER_POD} ${CONTAINER} > ${METALLB_PATH}/pods/${SPEAKER_POD}/${CONTAINER}/dump_frr
+      done
+    done
+      mkdir -p ${METALLB_PATH}/pods/${SPEAKER_POD}/speaker/
+
+      oc -n ${METALLB_NS} exec ${SPEAKER_POD} -c speaker -- sh -c "cat etc/frr_reloader/reloader.pid" > \
```

Comment:
Why we need both this and the next one?

Reply:
you're right we don't, I had a misunderstanding about the way the reloader works